src/ir/item.rs

```diff
@@ -825,6 +826,50 @@ impl Item {
             _ => None,
         }
     }
+
+    /// Returns whether the item is a constified module enum
+    fn is_constified_enum_module(&self, ctx: &BindgenContext) -> bool {
```

Comment:
Hmm... I see. I guess I'm not sure why would making an alias claim it's a constified enum module be necessary.

I guess I'll poke a bit at the code this evening when I have the chance and see which tests break with that.